External/FEXCore/Source/Interface/Core/ArchHelpers/MContext.h

```diff
@@ -13,6 +13,11 @@
 
 namespace FEXCore::ArchHelpers::Context {
 
+enum ContextFlags {
```

Comment:
Perhaps

```suggestion
enum ContextFlags : uint32_t {
```

To give it a concrete size and allow it to be forward declared

Reply:
Resolved.